tests/OptionalTest.php

```diff
@@ -296,6 +295,28 @@ public function orElseThrowTheEmptyValue()
         $result = $empty_optional->orElseThrow('\\BadFunctionCallException');
     }
 
+    /**
+     * @test
+     * @expectedException RuntimeException
+     * @expectedExceptionMessage '\ArrayObject' not implement Throwable.
+     */
+    public function orElseThrowTheNotThrowable()
```

Comment:
__[phpmd]__
* The variable $empty_optional is not named in camelCase.
* The variable $empty_optional is not named in camelCase.